src/convertToObject.js

```diff
@@ -12,7 +12,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleCss = {};
+
+  const filteredString = sourceString
```

Comment:
Just create more meaningful variable name and don't use data type name in variable names